keyboards/handwired/owlet60/keymaps/oled_testing/rules.mk

```diff
@@ -0,0 +1,48 @@
+SRC += matrix.c
```

Comment:
You don't need all of this here.  

In fact, the only thing you probably need in this file is:

```
OLED_DRIVER_ENABLE = yes
```

Reply:
Thank you for pointing that out. I've deleted all the redundant rules on my latest commit.

Reply:
Is there anything else i need to fix with my code? Thank you